reinforcement_learning/rl_unity_ray/rl_unity_ray.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Add a link to Unity. It'd be also useful to explain ml-agent as it's used as a Gym wrapper.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>A link to Unity or Unity ml-agent?</p>

Reply:
I suggest both. Essentially we want to explain the methodology briefly so that the readers won't get confused between our example vs. what rllib has (https://github.com/ray-project/ray/blob/master/rllib/examples/unity3d_env_local.py). As to my understanding, in our case Unity is the underlying game engine, Barracuda is the inference engine, and ml-agent is the toolkit that provides the bundle. Also consider including in README.

Reply:
Our example is essentially overlapped with what rllib offers.

Reply:
It'd be useful to explain both the overlaps and the differences if any (custom binary?), especially the relationship between our entrypoint and `unity3d_env_local.py`.

Reply:
I'll say not much difference. They indeed have better support than us. We'll migrate to that after this.